client/OPUNetGameSelectWnd.cpp

```diff
@@ -212,6 +204,20 @@ void OPUNetGameSelectWnd::OnInit()
 	timer = SetTimer(this->hWnd, 0, timerInterval, nullptr);
 }
 
+void OPUNetGameSelectWnd::InitializeMaxPlayersComboBox()
+{
+	char buffer[MaxServerAddressLen];
+
+	// Add the MaxPlayer options
+	for (int i = 2; i <= 6; ++i)
+	{
+		// Add the number of player to the combo box
+		scr_snprintf(buffer, sizeof(buffer), "%i", i);
+		SendDlgItemMessage(this->hWnd, IDC_MaxPlayers, CB_ADDSTRING, 0, (LPARAM)buffer);
```

Comment:
Going beyond reducing the `buffer` size from `MaxServerAddressLen` to `2`, you can get rid of `buffer` entirely by using `std::to_string` instead of `scr_snprintf`.